test/assertions_test.rb

```diff
@@ -124,7 +124,7 @@ def test_assert_statsd_call
       end
     end
 
-    assert_assertion_triggered do
+    assert_no_assertion_triggered do
```

Comment:
I think a subset makes sense. :+1:

If you end up sending more tags, DD always aggregates over them by default anyway. So the presence of additional tags should not break your monitors or dashboard by sending additional tags.